Dockerfile

```diff
@@ -1,4 +1,16 @@
+ARG BUILDER_IMAGE=golang:1.18-buster
 ARG BASE_IMAGE=alpine:3.15
+
```

Comment:
this will break the goreleaser

Reply:
why?

Reply:
becasue goreleaser uses the binary that is built to inject that in the image, and this update will try to run the full dockerfile but goreleaser does not expected that https://goreleaser.com/customization/docker/

Reply:
got it